src/target/startup.tcl

```diff
@@ -203,7 +203,6 @@ proc init_target_events {} {
 	foreach t $targets {
 		set_default_target_event $t gdb-flash-erase-start "reset init"
 		set_default_target_event $t gdb-flash-write-end "reset halt"
-		set_default_target_event $t gdb-attach "halt"
```

Comment:
I don't know what exactly happens in this file, but changing some generic startup that applies to all targets is impossible to test for regressions. We shouldn't do that. Even though I agree this "halt" should be a nop.

Reply:
Makes sense.  I don't remember exactly why I did this, but it was probably some request to not implicitly halt under all conditions.

Reply:
Keep the original version of this file.